coins/testnet/ltz.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "litecoinz_testnet",
+    "symbol": "ltzt",
+    "algorithm": "equihash",
+    "overwinter": true,
```

Comment:
If I'm not mistaken, sapling is a newer version than overwinter so you only need sapling, not both.